vm/vmExpr.go

```diff
@@ -560,6 +560,19 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 		}
 		return rhsV, nil
 
+	case *ast.IfInvalidOpExpr:
+		lhsV, err := invokeExpr(e.Lhs, env)
+		isZeroValue := reflect.DeepEqual(lhsV.Interface(), reflect.Zero(lhsV.Type()).Interface())
+		isEmpty := (lhsV.Kind() == reflect.Slice || lhsV.Kind() == reflect.Map) && lhsV.Len() == 0
```

Comment:
isEmpty really nil?

Reply:
It's not in Go, but from a scripting perspective, it's usually the case that an empty slice or a map can be checked in scripting languages with the equivalent of:
if !variable {
}